README.md

```diff
@@ -40,6 +40,30 @@ Table references and indexes in a migration will be applied to the same tenant p
 Apartmentex.new_tenant(Repo, tenant)
 ```
 
+When you need to update a tenant's schema based on new migrations, you can run:
+
+```elixir
+# Runs all migrations necessary for the tenant, based on that tenant's
+`schema_migrations` table
+# Returns a tuple that is either:
+# {:ok, prefix_of_tenant, versions_migrated}
+# {:error, prefix_of_tenant, db_error_message}
+
+{status, prefix, versions_or_error} = Apartmentex.migrate_tenant(Repo, tenant)
```

Comment:
@Dania02525 updated in https://github.com/Dania02525/apartmentex/pull/14/commits/2877fe4672a19b89024b0d7197e49b68b00247ad to remove the `rollback_tenant` function, and pass through options directly to `Ecto.Migrator.run`